student.rb

```diff
@@ -4,6 +4,7 @@ class Student < Person
   def initialize(classroom, age, name = 'Unknown', parents_permission = 'true')
     super(age, name, parents_permission)
     @classroom = classroom
+    @classroom.add_student(self)
   end
 
```

Comment:
- [ ] Kindly make sure to set up the Setters and Getters for `name`, `:age`, and `:rentals` so they can be accessible. you can do this by adding them to `attr_accessor`

Reply:
Bro, I've repeated this week and this Branch was approved in the previous week and it's solo activity!

Reply:
@Fahim-Khan-P, I see. but you didn't set up the proper relationships in the `Student` Class, also your class should have setters and getters. Maybe the previous reviewer forgot to mention this.
This is a repeated project so I should review it as a first review, not a re-review.

Reply:
@Ismailco, Thanks for your suggestion, as you instructed the given changes in a descriptive manner so I added them in the next step of my project, I mean that I implemented it in the **Basic UI** step of this project.

**Once again special thanks for your beneficial advice!**